### ISSUE TITLE
Demo web cannot signUp and  login

### ISSUE THREAD

**Issue by UlyC** [ddanielsantos/cloneboxd#34]
Cannot preview other pages even if I'm not logged in

**Comment by ddanielsantos** [comment]
I tried to access ``signUp`` and ``login`` in the demo without been logged and had success both in the Production and in the Preview environment, the only pages that require to be authenticated are: ``Home (/)``, ``Review (/review)`` and ``SearchMovie (/search-movie)`` (only exists in the Preview).

Are you still facing this error?

**Comment by UlyC** [comment]
Yes, I want to view these pages:   Home (/), Review (/review) and SearchMovie (/search-movie).
But it is necessary to log in first.
Then I can access `signUp `and `login` page. But I can't create an account or log in.
When I try to create an account, it response 400 Bad Request:
![image](https://user-images.githubusercontent.com/36838240/175926185-17800770-5670-43ed-ad62-b4826ee78b19.png)

**Comment by ddanielsantos** [comment]
As soon as I have some spare time, I'll check this, thank you 🤝
